src/applications/check-in/components/LanguagePicker.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import withTranslationEnabled from '../containers/withTranslationEnabled';
+
+function LanguagePicker() {
+  const { i18n } = useTranslation();
+
+  function changeLanguage(e) {
+    i18n.changeLanguage(e.target.value);
+  }
+
+  function buttonClass(language) {
```

Comment:
We could potentially make this "if elseif else" block a switch statement if we anticipate adding several more lanaguages in the future. It's also perfectly fine as is though.

Reply:
Duplicate comment -- marking resolved